### DISCUSSION TITLE
the monthly amount of forks per day I can do?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#52730] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

I would like to know the monthly amount of forks per day that I can do. 
- the monthly amount of forks per day I can do? 
- Are there any monthly, daily, weekly, yearly fork limits?

**Comment by jge162** [comment, marked as answer]
I do not believe there is a limit. It’s a key component of GitHub.
